patches/chrome-browser-resources-settings-page_visibility.js.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/chrome/browser/resources/settings/page_visibility.js b/chrome/browser/resources/settings/page_visibility.js
+index 419bd55a6866a6051e3b1a67443830e47c99e391..ab89258f4239a1040d70af7a31b64738c86f1e30 100644
+--- a/chrome/browser/resources/settings/page_visibility.js
++++ b/chrome/browser/resources/settings/page_visibility.js
+@@ -125,6 +125,31 @@ cr.define('settings', function() {
+       multidevice: true,
+     };
+     // </if>
++    // <if expr="not _google_chrome">
++    pageVisibility = {
```

Comment:
I think it would be better to specifically override things here:
`pageVisibility. a11y = false` for instance

Reply:
or if we want our own completely independent list then maybe we should override IDR_SETTINGS_PAGE_VISIBILITY_JS ?

Reply:
pageVisibility would be undefined for non chrome os && not guest.
And sadly I couldn't only write the false values here because there are places using hidden=[[!pageVisibility.people]] which make undesired items to be hidden if the value was undefined.

I'll try overriding IDR_SETTINGS_PAGE_VISIBILITY_JS and see how it goes first thing tomorrow.